docs/docusaurus/docs/guides/connecting_to_your_data/cloud/s3/components_spark/_add_csv_asset_to_spark_s3_datasource.md

```diff
@@ -1,4 +1,4 @@
-Add a CSV `Asset` into your `Datasource` by using the `add_csv_asset` function.
```

Comment:
We should probably take Data Source out of the backticks indicating code.  In the code itself, the class name is `Datasource`.  If we are using Data Source instead it isn't correct to indicate that it is the name in the code.  This will probably take just a moment with a find/replace.

Reply:
@Rachel-Reverie there are no instances of _Data Source_ within backticks indicating code. This was resolved with your previous edit request.